client/app/html/index.html

```diff
@@ -5,6 +5,7 @@
     <title>Weave Scope</title>
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script language="javascript">window.__WEAVEWORKS_CSRF_TOKEN = "$__CSRF_TOKEN_PLACEHOLDER__";</script>
```

Comment:
@fbarl index.html can't access our constants because it would have to load the bundle first.